.github/stale.yml

```diff
@@ -8,7 +8,10 @@ daysUntilStale: 14
 daysUntilClose: false
 
 # Issues or Pull Requests with these labels will never be considered stale. Set to `[]` to disable
```

Comment:
Maybe we update the wording to clarify the exemption list, since the PRs with `Lifecycle/2:inactive` and `Lifecycle/3:orphaned` labels are stale but we want the bot to ignore them.

```
Issues or Pull Requests with these labels will be ignored by the stale bot. Set to `[]` to disable
```